### PR TITLE
Added More Wanted Level Effects

### PR DESCRIPTION
**Pull request by OnlyRealNubs**
- 1 Wanted Star
- 3 Wanted Stars
- Fake Wanted Level

### CODE REVIEW

File: ChaosMod/Effects/db/Player/PlayerWantedController.cpp [hunk 1]
```diff
@@ -49,4 +52,111 @@ REGISTER_EFFECT(nullptr, nullptr, OnTickNeverWanted, EffectInfo
...
+REGISTER_EFFECT(OnStartOne, nullptr, nullptr, EffectInfo
+	{
+		.Name = "1 Wanted Star",
+		.Id = "player_1star",
+		.IncompatibleWith = { "player_neverwanted" }
+	}
+);
+// clang-format on
+
+static enum WantedLevelType
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Player/PlayerWantedController.cpp:94]:
```suggestion
enum WantedLevelType
```

File: ChaosMod/Effects/db/Player/PlayerWantedController.cpp [hunk 2]
```diff
@@ -49,4 +52,111 @@ REGISTER_EFFECT(nullptr, nullptr, OnTickNeverWanted, EffectInfo
...
+};
+
+struct FakeWantedInfo
+{
+	std::string szName;
+	WantedLevelType eWantedType;
+	int iStars;
+};
+
+std::vector<FakeWantedInfo> m_fakeWantedLevels =
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Player/PlayerWantedController.cpp:107]:
```suggestion
static std::vector<FakeWantedInfo> m_fakeWantedLevels = 
```

**Review by pongo1231 — CHANGES_REQUESTED** [review]

**Comment by OnlyRealNubs** [comment]
Resolved. Lmk if i did something wrong

**Comment by pongo1231** [comment]
LGTM

**Pull request merged**